source/shared/net/concepts/concepts-delegation.rst

```diff
@@ -0,0 +1,54 @@
+
+The second cap is on the amount of stake in a pool with respect to the stake of the baker. For example, the total stake of a pool can be at most 3 times the stake of the baker. This bound is for security reasons as it prevents a party with a small stake from controlling too large a part of the system. Every baker’s power is thus still proportional to their own stake, which is crucial for cryptographic security to hold.
+
+These two caps are soft caps in the sense that a pool can have more stake than allowed, but then at most the cap value will be taken into account in the consensus algorithm and when distributing rewards. This means that a pool violating this bound will gradually receive less rewards per CCD in the pool as the size of the pool continues to increase.
+
+Pool reliability
+----------------
+
+The rewards earned by a pool depend greatly on the reliability of the baker. If they miss a slot where they were chosen to bake, if they go offline, or if they do not follow the protocol and get jailed, then less rewards will be distributed to the entire pool. To help a delegator choose the best pool possible, statistics are available on `CCDScan <https://ccdscan.io>`_ about the performance of the different pools, the reliability of the baker, and the size of the pool.
+
+Bakers cannot choose the commission values themselves. There is a fixed commission value that is the same for all pools. This is to favor healthy competition between pools based on the quality of their baking, and not have some dishonest parties try to increase their control over the system by setting commissions to 0 to attract more delegators.
+
+Passive delegation
+------------------
+
+For CCD holders who do not want to regularly check the performance of their pool, but just want a safe way of earning interest, passive delegation offers a low-risk low-reward alternative. Passive delegation is not associated with a specific baker; it can be thought of as distributing its capital to each pool in proportion to the pool's stake. It is not affected by the poor performance of a single baker. But the parameters are set in such a way that a party delegating to passive delegation earns less than by delegating to a reliable baker.
+
+Time and cool-downs
+===================
+
+Changes to the pools are performed every 24 hours. So opening a pool, increasing the stake, moving the stake between pools or between passive delegation and a baking pool all occur at the payday. At payday, rewards gathered over a 24 hour period are distributed at the same time. If, however, you make a change in delegation in the last epoch before pay day, then the change has to wait until the second pay day.
```

Comment:
I think we spelled pay day as two words in the mobile wallet. Is that incorrect? Here it's in one word

Reply:
I've fixed it here and in the glossary now. Should be pay day everywhere.